src/Mod/Mesh/Gui/Workbench.cpp

```diff
@@ -125,9 +125,9 @@ class MeshInfoWatcher : public Gui::TaskView::TaskWatcher, public Gui::Selection
         if (countPoints > 0) {
             numPoints->setText(QString::number(countPoints));
             numFacets->setText(QString::number(countFacets));
-            numMin->setText(QString::fromLatin1("X: %1\tY: %2\tZ: %3")
+            numMin->setText(tr("X: %1\tY: %2\tZ: %3")
                 .arg(bbox.MinX).arg(bbox.MinY).arg(bbox.MinZ));
-            numMax->setText(QString::fromLatin1("X: %1\tY: %2\tZ: %3")
+            numMax->setText(tr("X: %1\tY: %2\tZ: %3")
```

Comment:
Does it really make sense to translate this?

Reply:
Perhaps for European languages ​​it does not make sense. do you know how, for example, the Arabs write this kind of line? right to left or left to right?